### PR TITLE
Fixes PagerInterface signatures

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: system/Pager/Pager.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ public function __construct($config, RendererInterface $view)
 	 *
 	 * @return string
 	 */
-	public function links(string $group = 'default', string $template = 'default_full'): string
+	public function links(string $group = null, string $template = 'default_full'): string
```
Comment (samsonasik) [inline system/Pager/Pager.php]: if this set to "default", in interface should be "default" too

File: system/Pager/PagerInterface.php [hunk 2]
```diff
@@ -10,19 +10,19 @@
 	 *
 	 * @return string
 	 */
-	public function links(string $template = 'default', string $group = null): string;
+	public function links(string $group = null, string $template = 'default'): string;
```
Comment (samsonasik) [inline system/Pager/PagerInterface.php]: inverted

Comment (samsonasik) [inline system/Pager/PagerInterface.php]: should $group value be a "default" or null ?

Comment (lonnieezell) [inline system/Pager/PagerInterface.php]: Should be 'default'

Comment (samsonasik) [inline system/Pager/PagerInterface.php]: done

**Comment by lonnieezell** [comment]
Great. Thanks!

**Pull request merged**